src/views/v2/commerce/checkout.php

```diff
@@ -55,6 +55,7 @@ class="tribe-tickets__commerce-checkout"
 			]
 		);
 		?>
+		<?php $this->template( 'checkout/purchaser-info' ); ?>
 		<?php $this->template( 'checkout/cart/empty' ); ?>
```

Comment:
```suggestion
		<?php $this->template( 'checkout/cart/empty' ); ?>
		<?php $this->template( 'checkout/purchaser-info' ); ?>
```

Let's group the cart stuff one after the other, and then include the `purchaser-info`